warehouse/models/intermediate/gtfs/gtfs_quality/int_gtfs_quality__scheduled_trips_in_tu_feed.sql

```diff
@@ -36,15 +36,15 @@ joined AS (
     AND quartet.trip_updates_gtfs_dataset_key IS NOT null
 
     LEFT JOIN fct_daily_scheduled_trips AS scheduled_trips
-      ON idx.date = scheduled_trips.service_date
+      ON idx.date = scheduled_trips.activity_date
      AND quartet.associated_schedule_gtfs_dataset_key = scheduled_trips.gtfs_dataset_key
 
     LEFT JOIN fct_observed_trips AS observed_trips
       ON idx.date = observed_trips.dt
      AND quartet.associated_schedule_gtfs_dataset_key = observed_trips.schedule_to_use_for_rt_validation_gtfs_dataset_key
      AND scheduled_trips.trip_id = observed_trips.trip_id
 
-     GROUP BY 1,2
+     GROUP BY 1, 2
```

Comment:
Ah lol I had to make these linting changes in #2304 too. (All good to leave them here but one of us will have to merge / rebase off the other eventually depending on who merges to main first)